modular_skyrat/modules/bonecrusher/bone_crusher.dm

```diff
@@ -0,0 +1,193 @@
+///***BONE CRUSHER***///
+//So basically this is shamelessly copy pasta'd crusher code because I cant figure out a good way to remove a component thats already been
+//initialized by the game, in this case the primary reason would be the adjustment of force_wielded from 20 to 15. I cant exactly figure out
+//how to do this so if someone has a good idea, drop it in the comments and I'll commit it. The only actual changes to this should be on
+//lines 11-16, 36, 38 and lines 48-49. Changes are noted where made.
+//Also yes this is recipe locked to ashwalkers only, since thats the primary demographic this is intended for. Intended to be slightly weaker
+//than the PKC, but better than most weapons ashwalkers can make because of trophies existing. Credit to the original owner of this code.
+
+
+/obj/item/bone_crusher
+	icon = 'modular_skyrat/modules/bonecrusher/bone_crusher.dmi'
+	icon_state = "bone_crusher"
+	inhand_icon_state = "crusher0"
+	lefthand_file = 'modular_skyrat/modules/bonecrusher/bone_crusher_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/bonecrusher/bone_crusher_righthand.dmi'
+	name = "makeshift proto-kinetic axe"
+	desc = "Upon closer examination, this appears to be what could be called a definitive 'tech club', compared to its estranged cousin, the Proto-Kinetic Crusher. Resembling more akin to a bone axe with metal strapped to it, this crude but effective weapon is the Ashwalkers response to the PKC."
+	force = 0 //You can't hit stuff unless wielded
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK
+	throwforce = 5
+	throw_speed = 4
+	armour_penetration = 10
+	custom_materials = list(/datum/material/iron=1000, /datum/material/bone = 12000)
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("smashes", "crushes", "cleaves", "chops", "pulps")
+	attack_verb_simple = list("smash", "crush", "cleave", "chop", "pulp")
+	sharpness = SHARP_EDGED
+	actions_types = list(/datum/action/item_action/toggle_light)
+	obj_flags = UNIQUE_RENAME
+	light_system = MOVABLE_LIGHT
+	light_range = 5
+	light_on = FALSE
+	var/list/trophies = list()
+	var/charged = TRUE
+	var/charge_time = 20 // Original value: 15
+	var/detonation_damage = 50
+	var/backstab_bonus = 25 // Original value: 30
+	var/wielded = FALSE // track wielded status on item
+
+/obj/item/bone_crusher/Initialize(mapload)
+	. = ..()
+	RegisterSignal(src, COMSIG_TWOHANDED_WIELD, .proc/on_wield)
+	RegisterSignal(src, COMSIG_TWOHANDED_UNWIELD, .proc/on_unwield)
+
+/obj/item/bone_crusher/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 50, 125) //Slightly faster and more efficient, original value: 60, 110
+	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=15) //Slightly weaker when wielded, original value: 20
```

Comment:
I cannot be approving code with blatant copy-paste when the value of `force_wielded` could be a variable stored on the crusher itself.

You can do much better than this, and it would save a lot of copy-paste, and would make the overall code much cleaner.

Reply:
Half the reason it was fully copy pasted instead of made a subtype was for the freedom of being able to make more intensive changes to the related code without having to add a bunch of variables, and without directly modifying PKC code itself. This is because of the crushers ComponentInitialize() in the parent. Im not sure how nicely this would play codewise if I changed out force_wielded = 15 to force_wielded = [var name here] and set the var above it in the item for both the PKC and the Bone Crusher.